### ISSUE TITLE
Appstream Stores support for chaotic-aur packages

### ISSUE THREAD

**Issue by SShubham1** [chaotic-aur/toolbox#39]
Is it possible to support appstream for chaotic-aur packages so that user could see the app infomation on software centre which uses appstream backend (e.g KDE Discover, Gnome Software etc.) and installs the package from it.

**Comment by PedroHLC** [comment]
@dr460nf1r3 @JustTNE these would need to be patched to use our repo: https://github.com/archlinux/svntogit-packages/blob/packages/archlinux-appstream-data/trunk/PKGBUILD

**Comment by SShubham1** [comment]
Maybe from `/usr/share/icons/hicolor/*/apps`, `/usr/share/pixmaps` and `/usr/share/icons/*.*` in the file listing of the package

**Comment by dr460nf1r3** [comment]
This would surely be an idea, but it would require installing every single package to extract the icons?

**Comment by SShubham1** [comment]
Yeah think so as it would requires a script to download all packages which has an icon and desktop file from repository.
